src/routes/popup/unlock.tsx

```diff
@@ -36,8 +32,6 @@ export default function Unlock() {
         duration: 2200
       });
     }
-
-    push("/");
```

Comment:
No need for a redirect as the Auth popup observes storage changes for `decryptionKey`, so it will automatically show the Unlock page if the wallet is locked and close the Auth popup once the wallet is unlocked.

Reply:
👍